massdriver-application-azure-function-app/main.tf

```diff
@@ -0,0 +1,85 @@
+locals {
+  max_length           = 24
+  storage_account_name = substr(replace(var.name, "/[^a-z0-9]/", ""), 0, local.max_length)
+}
+
+module "application" {
+  source                      = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=22d422e"
+  name                        = var.name
+  service                     = "function"
+  application_identity_id     = azurerm_linux_function_app.main.identity[0].principal_id
+  create_application_identity = false
```

Comment:
Can we add a comment about why we don’t need an app identity made by this module

Reply:
Sure thing

Reply:
Done